picard/tagger.py

```diff
@@ -345,30 +359,36 @@ def move_files(self, files, target):
         elif isinstance(target, ClusterList):
             self.cluster(files)
 
-    def add_files(self, filenames, target=None):
+    def add_files(self, list_of_filenames, target=None):
         """Add files to the tagger."""
-        ignoreregex = None
-        pattern = config.setting['ignore_regex']
-        if pattern:
-            ignoreregex = re.compile(pattern)
-        ignore_hidden = config.setting["ignore_hidden_files"]
-        new_files = []
+        # If we want to know the total number of files e.g. to warn the user
+        # sum the lengths of the lists.
+        # number_of_files = sum([len(x) for x in list_of_filenames])
+        for filenames in list_of_filenames:
+            thread.run_task(partial(self._open_files, filenames),
+                    partial(self._open_files_finished, target),
+                    priority=2,
+                    thread_pool=self.load_thread_pool)
+
+    def _open_files(self, filenames):
+        new_files = {}
         for filename in filenames:
             filename = os.path.normpath(os.path.realpath(filename))
-            if ignore_hidden and is_hidden(filename):
```

Comment:
@samj1912 Since I have fixed this review comment can you please mark it as fixed?